packages/vscode-extension/package.nls.json

```diff
@@ -267,6 +267,8 @@
     "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunnel service hasn't been started. Please wait for a moment or restart the local tunnel task.",
     "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit try to get the first HTTPS URL from %s and failed.",
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.launchTeamsWebClientError": "Failed to launch Teams web client. %s",
+    "teamstoolkit.localDebug.launchTeamsWebClientStoppedError": "Launch Teams web client task stopped with exit code '%s'.",
```

Comment:
either "Task to launch Teams web client ..." 
or "Launch-Teams-web-client task ..."

Reply:
Updated.